api/http/routing.go

```diff
@@ -232,11 +238,11 @@ func ReleasesRoutes(controller *DeploymentsApiHandlers) []*rest.Route {
 	}
 }
 
-func FMTConfigURL(hostname, deploymentID, deviceType, deviceID string) string {
+func FMTConfigURL(scheme, hostname, deploymentID, deviceType, deviceID string) string {
 	repl := strings.NewReplacer(
 		":deployment_id", deploymentID,
 		":device_type", deviceType,
 		":device_id", deviceID,
 	)
-	return fmt.Sprintf("https://%s%s", hostname, repl.Replace(ApiUrlDevicesDownloadConfig))
+	return fmt.Sprintf("%s://%s%s", scheme, hostname, repl.Replace(ApiUrlDevicesDownloadConfig))
```

Comment:
now I am sure the commits change the previous commits. the same comment: how about simple concat?

Reply:
Sure